src/screens/toDoList/ToDoList.tsx

```diff
@@ -0,0 +1,61 @@
+import React, {useMemo} from 'react';
+import {FlatList, Modal, StyleSheet, Text, View} from 'react-native';
+import ToDoListRow from './components/ToDoListRow';
+import FloatingButton from '../../ui/FloatingButton';
+import {useToDoList} from './useToDoList';
+import EditModal from '../../widgets/modal/EditModal';
+import {useModal} from '../../widgets/modal/useModal';
+
+export default function ToDoList() {
+  const {
+    toDoListArray,
+    handleAddItem,
+    handleDeleteItem,
+    openRow,
+    handleRowSwipe,
+  } = useToDoList();
+
+  const {modalVisible, openModal, closeModal} = useModal();
+
+  const buttonState = useMemo(() => {
+    return modalVisible ? '-' : '+';
+  }, [modalVisible]);
+
+  return (
+    <View style={styles.container}>
+      <Text style={styles.textHeader}>To Do List</Text>
+      <FlatList
+        keyExtractor={item => item.uuid}
+        data={toDoListArray}
+        renderItem={({item}) => {
+          return (
+            <ToDoListRow
+              toDoListItem={item}
+              deleteItem={() => handleDeleteItem(item)}
+              isOpen={openRow === item.uuid}
+              onSwipe={() => handleRowSwipe(item.uuid)}
+            />
+          );
+        }}
```

Comment:
Important, renderItem must be a static function. This means that it should be outside the component and not be an inline function